get-stack-resource.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
this is actually a `git mv` plus some changes... not sure why GH thinks its a new file...